llvm/lib/Target/X86/MCTargetDesc/X86BaseInfo.h

```diff
@@ -22,1299 +22,1286 @@
+  ///    SYMBOL_LABEL @TPOFF
+  MO_TPOFF,
+  /// MO_DTPOFF - On a symbol operand this indicates that the immediate is
+  /// the offset of the GOT entry with the TLS offset of the symbol. Used
+  /// in the local dynamic TLS access model.
+  /// See 'ELF Handling for Thread-Local Storage' for more details.
+  ///    SYMBOL_LABEL @DTPOFF
+  MO_DTPOFF,
+  /// MO_NTPOFF - On a symbol operand this indicates that the immediate is
+  /// the negative thread-pointer offset for the symbol. Used in the IA32
+  /// local exec TLS access model.
+  /// See 'ELF Handling for Thread-Local Storage' for more details.
+  ///    SYMBOL_LABEL @NTPOFF
+  MO_NTPOFF,
+  /// MO_GOTNTPOFF - On a symbol operand this indicates that the immediate is
+  /// the offset of the GOT entry with the negative thread-pointer offset for
+  /// the symbol. Used in the PIC IA32 initial exec TLS access model.
+  /// See 'ELF Handling for Thread-Local Storage' for more details.
+  ///    SYMBOL_LABEL @GOTNTPOFF
+  MO_GOTNTPOFF,
+  /// MO_DLLIMPORT - On a symbol operand "FOO", this indicates that the
+  /// reference is actually to the "__imp_FOO" symbol.  This is used for
+  /// dllimport linkage on windows.
+  MO_DLLIMPORT,
+  /// MO_DARWIN_NONLAZY - On a symbol operand "FOO", this indicates that the
+  /// reference is actually to the "FOO$non_lazy_ptr" symbol, which is a
+  /// non-PIC-base-relative reference to a non-hidden dyld lazy pointer stub.
+  MO_DARWIN_NONLAZY,
+  /// MO_DARWIN_NONLAZY_PIC_BASE - On a symbol operand "FOO", this indicates
+  /// that the reference is actually to "FOO$non_lazy_ptr - PICBASE", which is
+  /// a PIC-base-relative reference to a non-hidden dyld lazy pointer stub.
+  MO_DARWIN_NONLAZY_PIC_BASE,
+  /// MO_TLVP - On a symbol operand this indicates that the immediate is
+  /// some TLS offset.
+  /// This is the TLS offset for the Darwin TLS mechanism.
+  MO_TLVP,
+  /// MO_TLVP_PIC_BASE - On a symbol operand this indicates that the immediate
+  /// is some TLS offset from the picbase.
+  /// This is the 32-bit TLS offset for Darwin TLS in PIC mode.
+  MO_TLVP_PIC_BASE,
+  /// MO_SECREL - On a symbol operand this indicates that the immediate is
+  /// the offset from beginning of section.
+  /// This is the TLS offset for the COFF/Windows TLS mechanism.
+  MO_SECREL,
+  /// MO_ABS8 - On a symbol operand this indicates that the symbol is known
+  /// to be an absolute symbol in range [0,128), so we can use the @ABS8
+  /// symbol modifier.
+  MO_ABS8,
+  /// MO_COFFSTUB - On a symbol operand "FOO", this indicates that the
+  /// reference is actually to the ".refptr.FOO" symbol.  This is used for
+  /// stub symbols on windows.
+  MO_COFFSTUB,
+};
+
+enum : uint64_t {
+  //===------------------------------------------------------------------===//
+  // Instruction encodings.  These are the standard/most common forms for X86
+  // instructions.
+  //
+  /// PseudoFrm - This represents an instruction that is a pseudo instruction
```

Comment:
The loss of blank lines in this seems worse in some places

Reply:
Do you mean the readability?

Reply:
Yes

Reply:
We can add some blank lines back if it helps.